### ISSUE TITLE
[Bug] Some UI elements are vertical now

### ISSUE THREAD

**Issue by Kaiser-Indrasil** [tsunamayo/Starship-EVO#3238]
Ver. 47A

Happens both to player speed buttons and Jetpack toggle.

![image](https://user-images.githubusercontent.com/63756057/99561887-5ff02c80-29bf-11eb-9b3b-1dccce364d3f.png)

**Comment by tsunamayo** [comment]
Lol I see, how I am suppose to make them fit lol... I might deactivate the hint if you make it custom... I wonder how they do that in space Engineer. thx

**Comment by ProPeach** [comment]
Maybe rename Mouse1 to MB1? Wonder if Joystick can be JS1 or similar

**Comment by amplified1** [comment]
consoles tend to show their buttons with icons of the actual button so just having a circle with a differently shaded circle inside of it could be better for a joystick

**Comment by tsunamayo** [comment]
yeah a logo will be better. Console tend to not have bindable input, console are much sweeter in a lot of aspects dev-wise ;)
